i3_agenda/__init__.py

```diff
@@ -81,7 +82,7 @@ def getEvents(service, allowed_calendars_ids):
     all = []
     for id in calendar_ids:
         events_result = service.events().list(calendarId=id, timeMin=now,
-                                            maxResults=10, singleEvents=True,
+                                            maxResults=args.maxres, singleEvents=True,
```

Comment:
This wouldn't work, wouldn't it? args is not declared in this function, it's declared only in the main function

Reply:
sorry for that
please review changes